content/en/docs/reference/kubectl/resource_printing/describe.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "Describe"
+linkTitle: "Describe"
+weight: 4
+type: docs
+description: >
+    Describes current resources
+---
+
+
+{{< alert color="success" title="TL;DR" >}}
+- Print verbose debug information about a Resource
+{{< /alert >}}
+
+# Describe Resources
```

Comment:
Thanks.
We've realized that a straight conversion isn't right. This proposed tweak should help a lot though:
```suggestion
```